lib/jsonapi/plugs/query_parser.ex

```diff
@@ -179,7 +186,7 @@ defmodule JSONAPI.QueryParser do
       if inc =~ ~r/\w+\.\w+/ do
         acc ++ handle_nested_include(inc, valid_includes, config)
       else
-        inc = inc |> dash() |> String.to_existing_atom()
+        inc = String.to_existing_atom(inc)
```

Comment:
@snewcomer here's an example of a backwards incompat change. I could add it back in with a deprecation warning.

Reply:
Since we are doing the whole shebang, probably ok.